src/components/playlisteditor/playlisteditor.ts

```diff
@@ -127,8 +127,6 @@ function triggerChange(select: HTMLSelectElement) {
 function populatePlaylists(editorOptions: PlaylistEditorOptions, panel: DialogElement) {
     const select = panel.querySelector<HTMLSelectElement>('#selectPlaylistToAddTo');
 
-    loading.hide();
```

Comment:
I wonder if `show` should be here (before the change). :thinking: 
If yes, it must go after `if (!select)`.

I just don't see which `show` corresponds to the `hide` below.

Reply:
Yeah that seems likely... although in my testing it is hidden before I see it anyway